peps/pep-0810.rst

```diff
@@ -948,7 +960,9 @@ A: The overhead is minimal:
 Benchmarking with the `pyperformance suite`_ shows the implementation is
 performance neutral when lazy imports are not used.
 
-.. _pyperformance suite: https://github.com/facebookexperimental/free-threading-benchmarking/blob/main/results/bm-20250922-3.15.0a0-27836e5/bm-20250922-vultr-x86_64-DinoV-lazy_imports-3.15.0a0-27836e5-vs-base.svg
+.. _pyperformance suite: https://github.com/facebookexperimental/
+    free-threading-benchmarking/blob/main/results/bm-20250922-3.15.0a0-27836e5/
+    bm-20250922-vultr-x86_64-DinoV-lazy_imports-3.15.0a0-27836e5-vs-base.svg
```

Comment:
This does [appear to work](https://pep-previews--4625.org.readthedocs.build/pep-0810/#faq), but is somewhat unorthodox, so I'd understand if you want to revert